packages/react-juce/src/components/Canvas.ts

```diff
@@ -59,13 +60,15 @@ export function bindCanvasContextProperties(ctx: any) {
 
 export interface CanvasProps {
   onMeasure?: (e: any) => void;
-  onDraw: (ctx: CanvasRenderingContext2D) => void;
+  onDraw: (ctx: CanvasRenderingContext2D, sizeState: object) => void;
   autoclear?: boolean;
+  ongoingTouches?: Array<Touch>;
```

Comment:
Could you clarify what `ongoingTouches` is for here? It feels like a thing that should not be part of Canvas's props API